src/main/java/seedu/duke/Ui.java

```diff
@@ -11,10 +11,10 @@
  * @since 2023-08-29
  */
 public class Ui {
-    public static final String i4 = "    ";
-    public static final String i5 = Ui.i4 + " ";
-    public static final String i7 = Ui.i5 + "  ";
-    public static final String line = Ui.i4 + "——————————————————————————————————————————————————————————————————";
+    public static final String I4 = "    ";
+    public static final String I5 = Ui.I4 + " ";
+    public static final String I7 = Ui.I5 + "  ";
```

Comment:
```suggestion
    public static final String INDENT_4 = "    ";
    public static final String INDENT_5 = INDENT_4 + " ";
    public static final String INDENT_7 = INDENT_5 +  "  ";`
```
I like how you make the indents into constants so it will easily be managed, however perhaps can consider a more intuitive constant name?